src/migrations/20231212094343-unleash-system-user.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+        ALTER TABLE users ADD COLUMN IF NOT EXISTS is_system BOOLEAN NOT NULL DEFAULT FALSE;
+        INSERT INTO users
+          (id, name, username, email, created_by, is_system) 
+        VALUES
+          (-1337, 'Used by unleash internally for performing system actions that have no user', 'unleash_system_user', 'system@getunleash.io', -1337, true);
```

Comment:
I'm a bit unsure about using such a long name in the UI. Does it break anything visually? We might just want a simpler name like "Unleash System" or something.

```suggestion
          (-1337, 'This is a system user utilized internally by Unleash to carry out automated system actions that do not require direct user intervention', 'unleash_system_user', 'system@getunleash.io', -1337, true);
```

Reply:
Thanks @nunogois that's a very good point!